### PR TITLE
(DOCSP-10760) - Fix Realm Node APIKey method

### PR DESCRIPTION
**Pull request by mongomoe**
Jira: https://jira.mongodb.org/browse/DOCSP-10760
Stage: https://jira.mongodb.org/browse/DOCSP-10760

### CODE REVIEW

File: source/node/authenticate.txt [hunk 1]
```diff
@@ -139,7 +139,7 @@ user's email address and password and pass it to ``App.logIn()``:
              console.error("Failed to log in", err);
            }
          }
-         loginEmailPassword("joe.jasper@example.com", "passw0rd").then(user => {
+         loginEmailPassword("cal.kestis@mailinator.com", "Bogano123!").then(user => {
```
Comment (cbush) [inline source/node/authenticate.txt:142]: I suggest not using real possible addresses 
```suggestion
         loginEmailPassword("cal.kestis@example.com", "Bogano123!").then(user => {
```

File: source/node/authenticate.txt [hunk 2]
```diff
@@ -161,7 +161,7 @@ user's email address and password and pass it to ``App.logIn()``:
...
```
Comment (cbush) [inline source/node/authenticate.txt:164]:
```suggestion
         loginEmailPassword("cal.kestis@example.com", "Bogano123!").then(user => {
```

File: source/node/authenticate.txt [hunk 3]
```diff
@@ -184,8 +184,8 @@ API key and pass it to ``App.logIn()``:
       .. code-block:: javascript
          
          async function loginApiKey(apiKey) {
-           // Create an anonymous credential
-           const credentials = Realm.Credentials.apiKey(apiKey);
+           // Create a APIKey credential
```
Comment (mongomoe) [inline source/node/authenticate.txt:187]:
```suggestion
           // Create an APIKey credential
```

File: source/node/authenticate.txt [hunk 4]
```diff
@@ -206,8 +206,8 @@ API key and pass it to ``App.logIn()``:
       .. code-block:: typescript
          
          async function loginApiKey(apiKey: string) {
-           // Create an anonymous credential
-           const credentials = Realm.Credentials.apiKey(apiKey);
+           // Create a APIKey credential
```
Comment (mongomoe) [inline source/node/authenticate.txt:209]:
```suggestion
           // Create an APIKey credential
```

**Pull request merged**